playbooks/base-settings.yaml

```diff
@@ -0,0 +1,75 @@
+---
+
+- name: Setup Base Image
+  hosts: all
+  become: yes
+  tasks:
+    - name: Set timezone to Europe/Vienna
+      community.general.timezone:
+        name: Europe/Vienna
+
+    - name: Install Chrony for time synchronization 
+      apt:
+        update_cache: yes
+        name: chrony
+
+    - name: Set default locale
+      ansible.builtin.debconf:
+        name: locales
+        question: locales/default_environment_locale
+        value: de_AT.UTF-8
+        vtype: select
+
+    - name: Set locales to be generated
+      ansible.builtin.debconf:
+        name: locales
+        question: locales/locales_to_be_generated
+        value: de_AT.UTF-8 UTF-8, en_US.UTF-8 UTF-8
+        vtype: select
+
+    - name: Configure keyboard
+      debconf:
+        name: console-data
+        question: console-data/keymap/full
+        value: de-latin1-nodeadkeys
+        vtype: select
+
+    - name: Add user olimex to i2c group
+      user:
+        name: olimex
+        append: true
+        groups: i2c
+
+    - name: Set static IP
+      # Only execute this task when the variable target_static_ip has been found
+      when: target_static_ip is defined        
+      copy:
+        dest: /etc/network/interfaces.d/eth0
+        content: |
+            allow-hotplug eth0
+
+            iface eth0 inet static
+                address {{ target_static_ip }}
+                netmask 255.255.0.0
+                broadcast 10.0.255.255
+                gateway 10.0.0.1
```

Comment:
Doesn't work for my network.

Reply:
Well, the point of this playbook is to provide a starting point for your own configurations, not to be universally applicable. So obviously, you'd have to change certain settings to make it work in your environment. I probably should not this in the README too.